planner/core/point_get_plan.go

```diff
@@ -1476,6 +1476,11 @@ func buildPointDeletePlan(ctx sessionctx.Context, pointPlan PhysicalPlan, dbName
 
 func findCol(tbl *model.TableInfo, colName *ast.ColumnName) *model.ColumnInfo {
 	for _, col := range tbl.Columns {
+		if colName.Name.L == model.ExtraHandleName.L && !tbl.PKIsHandle {
+			colInfo := model.NewExtraHandleColInfo()
+			colInfo.Offset = len(tbl.Columns) - 1
+			return colInfo
+		}
```

Comment:
This can be moved out of the for loop?

Reply:
You are right. Updated, PTAL @tiancaiamao